frontends/web/src/hooks/qrcodescanner.test.ts

```diff
@@ -1,75 +0,0 @@
-/**
- * Copyright 2023 Shift Crypto AG
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { MutableRefObject } from 'react';
-import { renderHook } from '@testing-library/react-hooks';
-import { BrowserQRCodeReader } from '@zxing/library';
-import { useQRCodeScanner } from './qrcodescanner';
-
-const mockedQRCodeReaderInstance = () => ({
-  getVideoInputDevices: jest.fn().mockResolvedValue(['camera1']),
-  decodeFromInputVideoDevice: jest.fn().mockResolvedValue({
-    getText: jest.fn().mockReturnValue('mockedQRValue')
-  }),
-  reset: jest.fn()
-});
-
-
-jest.mock('../components/alert/Alert', () => ({
-  alertUser: jest.fn()
-}));
-
-describe('useQRCodeScanner', () => {
-  it('should determine if a camera is available', async () => {
-    const qrCodeReaderRef = { current: mockedQRCodeReaderInstance() } as unknown as MutableRefObject<BrowserQRCodeReader | undefined>;
-    const props = {
-      qrCodeReaderRef,
-      activeScanQR: false,
-      onChangeActiveScanQR: jest.fn(),
-      parseQRResult: jest.fn()
-    };
-
-    const { result, waitForNextUpdate } = renderHook(() => useQRCodeScanner(props));
```

Comment:
@shonsirsha to get this test back it would need some virtual HTMLVideoElement I think.
  
![Screen Shot 2023-09-18 at 18 40 34](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/57ae2884-1850-4ae3-a743-d33f6c35e4d0)

Reply:
Yes please take the tests over 👍 . thank you 🙏

Reply:
but if not straight forward or if it is too annoying we can also drop.